CMakeLists.txt

```diff
@@ -393,6 +400,8 @@ ADD_CUSTOM_TARGET(updateversion ALL
 ################################################################################
 # Then add all other dependencies
 
+FILE(GLOB COMMON_SRC ${PROJECT_SOURCE_DIR}/commonSrc/*.cpp ${PROJECT_SOURCE_DIR}/commonSrc/*.h)
```

Comment:
don't use glob - if a file's missing the buildsystem does not recognize that and you get strange errors

Reply:
Yeah but we are using this already. It's just so cumbersome to add a new class without a glob. But yes maybe we should change it back. But at least I'd put it in an extra cmake file to keep it clean (we have 100s files or so)